RELEASE.md

```diff
@@ -0,0 +1,84 @@
+# Release Instructions.
```

Comment:
nitpick - do you need a period for a title?

Reply:
Nope - good catch.